### DISCUSSION TITLE
新建条目笔记后，单击“插入模板到光标行“没反应，调取不到模板

### DISCUSSION THREAD

**Discussion by PyQiaoJoe** [windingwind/zotero-better-notes#814] [Q&A]
今天突然发现，新建条目笔记后，单击“插入模板到光标行”没反应，调取不到模板，重启插件，重启zotero，重启计算机都不行，烦请各位大佬支支招，谢谢！（zotero 6）

**Comment by windingwind** [comment]
bug反馈请在issue提问，QA是使用技巧答问
